tutorials/io/plot_20_reading_eeg_data.py

```diff
@@ -161,45 +165,73 @@
 eXimia EEG data (.nxe)
 ======================
 
-EEG data from the Nexstim eXimia system can be read in using the
-:func:`mne.io.read_raw_eximia` function.
+EEG data from the Nexstim eXimia system can be read with
+:func:`mne.io.read_raw_eximia`.
 
 
 .. _import-persyst:
 
 Persyst EEG data (.lay, .dat)
 =============================
 
-EEG data from the Persyst system can be read in using the
-:func:`mne.io.read_raw_persyst` function.
+EEG data from the Persyst system can be read with
+:func:`mne.io.read_raw_persyst`.
 
-Note that not all the subject metadata may be properly read in
-due to the fact that Persyst changes its specification
-sometimes from version to version. Please submit an issue, or
-pull request if you encounter a problem.
+Note that not all subject metadata may be properly imported because Persyst
+sometimes changes its specification from version to version. Please let us know
+if you encounter a problem.
 
-Nihon Kohden EEG data (.EEG, .21E, .PNT, .LOG)
+
+Nihon Kohden EEG data (.eeg, .21e, .pnt, .log)
 ==============================================
 
 EEG data from the Nihon Kohden (NK) system can be read using the
 :func:`mne.io.read_raw_nihon` function.
 
 Files with the following extensions will be read:
 
-- The ``.EEG`` file contains the actual raw EEG data.
-- The ``.PNT`` file contains the metadata related to the recording, such
-  as the measurement date.
-- The ``.LOG`` file contains annotations for the recording.
-- The ``.21E`` file contains the channel and electrode
-  recording system information.
+- The ``.eeg`` file contains the actual raw EEG data.
+- The ``.pnt`` file contains metadata related to the recording such as the
+  measurement date.
+- The ``.log`` file contains annotations for the recording.
+- The ``.21e`` file contains channel and electrode information.
 
-Reading ``.11D``, ``.CMT``, ``.CN2``, and ``.EDF`` files is currently not
+Reading ``.11d``, ``.cmt``, ``.cn2``, and ``.edf`` files is currently not
 supported.
 
-Note that not all the subject metadata may be properly read in
-due to the fact that NK changes the specification
-sometimes from version to version. Please submit an issue, or
-pull request if you encounter a problem.
+Note that not all subject metadata may be properly read because NK changes the
+specification sometimes from version to version. Please let us know if you
+encounter a problem.
+
+
+XDF data (.xdf, .xdfz)
+======================
+
+MNE-Python does not support loading
+`XDF <https://github.com/sccn/xdf/wiki/Specifications>`_ files out of the box,
+mainly because XDF is an extremely flexible file format which supports signals
+from various modalities recorded with different sampling rates. However, it is
```

Comment:
sounds good in general, but I suggest a rephrasing to make it sound less like: "we don't provide it because it's so great" :slightly_smiling_face: 

```suggestion
MNE-Python does not support loading
`XDF <https://github.com/sccn/xdf/wiki/Specifications>`_ files out of the box,
because the inherent flexibility of the XDF format makes it difficult to provide a 
one-size-fits-all function.
For example, XDF supports signals from various modalities recorded with different
sampling rates. However, it is
```